doc/advanced-configuration/outgoing-connections.md

```diff
@@ -1,177 +1,92 @@
-# Outgoing connections
-
-MongooseIM can be configured to talk to external service like databases or HTTP servers in order to get or set the required data.
+MongooseIM can be configured to talk to external services like databases or HTTP servers in order to get or set the required data.
 The interface for outgoing connections management was unified and is now available via the `outgoing_pools` config option for the following type of connections:
 
 * `cassandra` - pool of connections to Cassandra cluster
 * `riak` - pool of connections to Riak cluster
 * `redis` - pool of connections to Redis server
-* `http` - pool of connections to various HTTP(S) servers MongooseIM can talk to, in example HTTP authentication backend or HTTP notifications
+* `http` - pool of connections to various HTTP(S) servers MongooseIM can talk to, for example HTTP authentication backend or HTTP notifications
 * `elastic` - pool of connections to ElasticSearch server
 * `rdbms` - pool of connections to an RDBMS database
 * `rabbit` - pool of connections to a RabbitMQ server
 * `ldap` - pool of connections to an LDAP server
-* `generic` - pool of generic workers not associated directly with a particular connection (SNS, PushNotifications)
-
-All the above pools are managed by [inaka/worker_pool](https://github.com/inaka/worker_pool) library.
+* `generic` - pool of generic workers not associated directly with a particular connection
 
-Every entry in the `outgoing_pools` is a 5-element tuple:
+* **Syntax:** Each pool is specified in a subsection starting with `[outgoing_pools.type.tag]`, where `type` is one of available connection types and `tag` is an arbitrary value.
```

Comment:
I think we should say here that the tag should uniquely identify the pool within each type - it's certainly not just an arbitrary value.